### PR TITLE
Add actions/labeler for auto label PRs

### PR DESCRIPTION
**Pull request by chenhunghan**
- Add [actions/labeler](https://github.com/actions/labeler). a Github Action automatically applying labels to PRs based on config defined in `.github/labeler-config.yml`

### CODE REVIEW

**Comment by nevalla** [comment]
We have also `./docs` folder in `extensions-docs` branch, so proposing adding that under `area/documentation`. `.azure-pipelines.yml` goes also under `area/ci`

File: .github/labeler-config.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+area/ui:
+  - src/renderer/**/*
+area/test:
+  - integration/**/*
+  - __mocks__/**/*
+area/extension:
+  - extensions/**/*
+  - src/extensions/**/*
+area/documentation:
+  - READEME.md
```
Comment (jakolehm) [inline .github/labeler-config.yml]:
```suggestion
  - README.md
  - docs/**/*
```

**Comment by chenhunghan** [comment]
> We have also `./docs` folder in `extensions-docs` branch, so proposing adding that under `area/documentation`. `.azure-pipelines.yml` goes also under `area/ci`

Added!

File: .github/labeler-config.yml (hunk 1 above)
Comment (chenhunghan) [reply .github/labeler-config.yml]: Fixed https://github.com/lensapp/lens/pull/1193/commits/63725864d3f216565df3184dac96a758454ce486

**Review by jakolehm — APPROVED** [review]
LGTM

**Pull request merged**